src/CoreWCF.Primitives/src/CoreWCF.Primitives.csproj

```diff
@@ -9,9 +9,11 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Hosting.Server.Abstractions" Version="2.1.1" />
     <PackageReference Include="Microsoft.AspNetCore.Http.Abstractions" Version="2.1.1" />
+    <PackageReference Include="Microsoft.Extensions.Caching.Memory" Version="2.1.1" />
     <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="2.1.1" />
```

Comment:
What is this being used for?

Reply:
This is used to keep the roles for same user https://github.com/CoreWCF/CoreWCF/pull/340/files#diff-0453087f6201cfcca1f04e109115308aebbb6dfb46fb806660071a9ef1c11d14R40